### PR TITLE
fixed documentation to say how to use frames option in runvx

### PR DESCRIPTION
**Pull request by paveltc**
Added that -frames:[number of frames]:ignore-eof needs to be used whenever wanting to repeat a gdf on the same input.

### CODE REVIEW

File: utilities/runvx/README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ If available, this project uses OpenCV for camera capture and image display.
       -frames:[<start>:]<end>|eof|live
           Run the graph/node for specified frames or until eof or just as live.
           Use live to indicate that input is live until aborted by user.
+          Use -frames:[amount of frames]:ignore-eof when you want to run on the same image multiple times.
```
Comment (rrawther) [inline utilities/runvx/README.md:36]: I would change to [number of frames] instead

**Review by rrawther — APPROVED** [review]
Added a comment. Please correct

File: utilities/runvx/README.md (hunk 1 above)
Comment (paveltc) [reply utilities/runvx/README.md]: I updated the PR. I changed to [number of frames] and I added the instructions for ignore-eof in runvx.cpp.
I will check the runvx code to see if more than just the ignore-eof command line argument needs to be changed to "loop"

Comment (paveltc) [reply utilities/runvx/README.md]: Changed ignore-eof to loop and put that in both the README.md and the runvx help.

**Comment by kiritigowda** [comment]
@paveltc - merge pending CI - http://math-ci.amd.com/blue/organizations/jenkins/mainline%2Fprecheckin%2FMIVisionX/detail/PR-1438/2/pipeline

Fix for #1157

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**